### DISCUSSION TITLE
Help using tailwindcss with Angular Universal

### DISCUSSION THREAD

**Discussion by julian-payne** [tailwindlabs/tailwindcss#11408] [Help]
I've followed [this documentation](https://tailwindcss.com/docs/guides/angular) minus the css part because I'm using scss so instead I did the below:
```scss
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import 'tailwindcss/utilities';
```

However the build result will not have tailwind styles applied.

I though about writing every single config I have for this project (tsconfig, angular.json, webpack.config, etc) but it would make this question very long so maybe I could receive some help from someone who actually have already solved this issue?

Further more, the project is in a NX workspace so that might be making things a little bit more complicated...

Any help, suggestion is well received.

**Comment by mybearworld** [comment]
What is your tailwind.config.js?

**Comment by computebender** [comment]
I am following the same setup and have the same issue. Minus using nx.

Using:
- Angular 17
- Tailwind 3.4

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./src/**/*.{html,ts}",
  ],
  theme: {
    extend: {
      fontFamily: {
        sans: ['Inter', 'sans-serif'],
      },
    },
  },
  plugins: [],
  corePlugins: {
    preflight: false,
   }
}
```
Note: Preflight is disabled because I am using https://www.npmjs.com/package/ngx-markdown and preflight removes its style.
